maboss/simulation.py

```diff
@@ -151,8 +151,12 @@ def get_maboss_cmd(self):
             pass
         elif l <= 128:
             maboss_cmd = "MaBoSS_128n"
-        else:
+        elif l <= 256:
             maboss_cmd = "MaBoSS_256n"
+        elif l <= 512:
+            maboss_cmd = "MaBoSS_512n"
+        else:
```

Comment:
shouldn't there be an assertion ensuring that `l <= 1024`?

Reply:
You're right, that's indeed cleaner.